frontend/src/components/OdhAppCard.tsx

```diff
@@ -67,20 +69,20 @@ const OdhAppCard: React.FC<OdhAppCardProps> = ({ odhApp }) => {
         target="_blank"
         rel="noopener noreferrer"
       >
-        {`Launch `}
+        Launch
```

Comment:
optional/future: removing the padding in-between copy and an icon, be good to have a snapshot of this output at some point to confirm the shift... we'll assume your visual confirmation was a-ok based on the updates to `OdhCard.scss`

Reply:
![image](https://user-images.githubusercontent.com/11633780/109959537-a428e380-7cb5-11eb-8f3a-4798903bcc44.png)